Sources/SimpleCurrencyConverter/ExchangeRatesNetworkManager.swift

```diff
@@ -27,8 +40,8 @@ final class ExchangeRatesNetworkManager {
             }
             
             var request = URLRequest(url: modifiedURL)
-            request.httpMethod = "GET"
-            request.setValue(apiKey, forHTTPHeaderField: "apikey")
+            request.httpMethod = Keys.httpMethod
+            request.setValue(apiKey, forHTTPHeaderField: Keys.httpHeader)
```

Comment:
`httpHeader` looks too generic. It erases context of this function. I'm thinking about having `Keys.Header.apiKey` instead